crates/miden-objects/src/account/code/mod.rs

```diff
@@ -285,6 +318,25 @@ impl Deserializable for AccountCode {
     }
 }
 
+// PRETTY PRINT
+// ================================================================================================
+
+impl PrettyPrint for AccountCode {
+    fn render(&self) -> vm_core::prettier::Document {
+        use vm_core::prettier::*;
+        let mut partial = Document::Empty;
+        let len_procedures = self.num_procedures();
+
+        for (index, printable_procedure) in self.get_printable_procedures().enumerate() {
+            partial += printable_procedure.render();
+            if index < len_procedures - 1 {
+                partial += nl();
+            }
+        }
```

Comment:
Related to the previous comment, I'd add the other procedure info here. Maybe it could look something like:
```
proc.{proc_root}
    storage.{offset}.{size}
    begin
        {proc body}
    end
end

proc.{proc_root}
    storage.{offset}.{size}
    begin
        {proc body}
    end
end
```